cellprofiler/gui/help.py

```diff
@@ -2027,287 +2027,323 @@
+        the File list panel.</p>
+    </li>
+    <li>In the <b>Metadata</b> module, check the box to enable metadata extraction. The key step here is to obtain the
+    metadata tags necessary to do two things:
+        <ul>
+            <li>Distinguish the movies from each other. This information is typically encapsulated in the filename
+            and/or the folder name.</li>
+            <li>For each movie, distinguish the timepoints from each other and ensure their proper ordering. This
+            information is usually contained in the filename.</li>
+        </ul>To accomplish this, do the following:
+        <ul>
+            <li>Select "{X_MANUAL_EXTRACTION}" or "{X_IMPORTED_EXTRACTION}" as the metadata extraction method. You will
+            use these to extract the movie and timepoint tags from the images.</li>
+            <li>Use "{X_MANUAL_EXTRACTION}" to create a regular expression to extract the metadata from the filename
+            and/or path name.</li>
+            <li>Or, use "{X_IMPORTED_EXTRACTION}" if you have a comma-delimited file (CSV) of the necessary metadata
+            columns (including the movie and timepoint tags) for each image. Note that microscopes rarely produce 
+            such a file, but it might be worthwhile to write scripts to create them if you do this frequently.</li>
+        </ul>If there are multiple channels for each movie, this step may need to be performed for each channel.
+        <p>In this example, you could do the following:</p>
+        <ul>
+            <li>Select "{X_MANUAL_EXTRACTION}" as the method, "From file name" as the source, and
+            <code>.*-(?P&lt;Well&gt;[A-P][0-9]{{2}})\.(?P&lt;Timepoint&gt;[0-9]{{3}})</code> as the regular expression.
+            This step will extract the well ID and timepoint from each filename.</li>
+            <li>Click the "Add" button to add another extraction method.</li>
+            <li>In the new group of extraction settings, select "{X_MANUAL_EXTRACTION}" as the method, "From folder
+            name" as the source, and <code>.*[\\/](?P&lt;Stain&gt;.*)[\\/].*$</code> as the regular expression. This
+            step will extract the stain name from each folder name.</li>
+            <li>Click the "Update" button below the divider and check the output in the table to confirm that the
+            proper metadata values are being collected from each image.</li>
+        </ul>
+    </li>
+    <li>In the <b>NamesAndTypes</b> module, assign the channel(s) to a name of your choice. If there are multiple
+    channels, you will need to do this for each channel.<br>
+        For this example, you could do the following:
+        <ul>
+            <li>Select "Assign images matching rules".</li>
+            <li>Make a new rule <code>[Metadata][Does][Have Stain matching][fluor]</code> and name it
+            <i>OrigFluor</i>.</li>
```

Comment:
@karhohs 
Could you suggest alternate names to replace OrigFluor and OrigFluo2? The current example is weird because renaming is done but it’s not substantive. Would be better if “fluor” is renamed to be “DNA” and “fluo2” becomes “actin”. But I don’t know if those are realistic, maybe you or Beth know the stain names better than I do.